spec.bs

```diff
@@ -492,8 +513,26 @@ The <dfn element-attr for=fencedframe>allow</dfn> attribute, when specified, det
 in the <{fencedframe}>'s [=fenced navigable container/fenced navigable=] is initialized. Its value
 must be a [=serialized permissions policy=]. [[!PERMISSIONS-POLICY]]
 
-The IDL attribute <dfn attribute for=HTMLFencedFrameElement>allow</dfn> must [=reflect=] the
-respective content attribute of the same name.
+The <dfn element-attr for=fencedframe>sandbox</dfn> attribute, when specified, enables a set of
+extra restrictions on any content hosted by the <{fencedframe}>. Its value must be an [=unordered
+set of unique space-separated tokens=] that are [=ASCII case-insensitive=]. The allowed values are
+the same as the allowed values for the <{iframe}>'s <{iframe/sandbox}> attribute.
+
+When an <{fencedframe}> element's <{fencedframe/sandbox}> attribute is set or changed while it has a
+non-null [=fenced navigable container/fenced navigable=], the user agent must [=parse a sandboxing
+directive=] given the attribute's value and the <{fencedframe}> element's [=fencedframe/fencedframe
+sandboxing flag set=].
+
+When an <{fencedframe}> element's <{fencedframe/sandbox}> attribute is removed while it has a
+non-null [=fenced navigable container/fenced navigable=], the user agent must empty the
+<{fencedframe}> element's [=fencedframe/fencedframe sandboxing flag set=].
```

Comment:
Make this more algorithmic by delegating to infra algorithms to clear a list, assuming it's all a list under the hood.

Reply:
It's not as clear as I would like it to be, but the [sandboxing flags set](https://html.spec.whatwg.org/multipage/browsers.html#sandboxing-flag-set) definition says it's a set, but "set" is unlinked. I'm going to assume it's an [ordered set](https://infra.spec.whatwg.org/#ordered-set), which is just a [list](https://infra.spec.whatwg.org/#list) under the hood, which means I can reference the [empty](https://infra.spec.whatwg.org/#list-empty) algorithm for it.

Reply:
That works, thanks!